modules/dmarc/dmarc.go

```diff
@@ -18,8 +18,11 @@ import (
 	"github.com/miekg/dns"
 	"github.com/zmap/zdns"
 	"github.com/zmap/zdns/modules/miekg"
+	"regexp"
 )
 
+const dmarcPrefixRegexp = "v[\x09\x20]*=[\x09\x20]*DMARC1[\x09\x20]*;[\x09\x20]*"
```

Comment:
Are \x09 and \x20 whitespace characters? Should we instead be using `[\s]*`?

Reply:
I used the ABNF from the [DMARC RFC](https://datatracker.ietf.org/doc/html/rfc7489) that states:

`dmarc-version   = "v" *WSP "=" *WSP %x44 %x4d %x41 %x52 %x43 %x31`

WSP comes from [ABNF RFC](https://datatracker.ietf.org/doc/html/rfc5234) and is defined as:

` WSP            =  SP / HTAB
                                ; white space`

`[\s]*` is not equal to `[\x09\x20]*` as the first one accepts also newline, form feed, and carriage return. [[1]](https://pkg.go.dev/regexp/syntax)

However, I now see that `"v"` should actually be case insensitive as ABNF strings are case insensitive [[ABNF 2.3]](https://datatracker.ietf.org/doc/html/rfc5234). So the code should rather look like this:

`const dmarcPrefixRegexp = "[vV][\x09\x20]*=[\x09\x20]*DMARC1[\x09\x20]*;[\x09\x20]*"`

What do you think?

Reply:
I agree with that - @dadrian you have any thoughts?

Reply:
Hard to argue against matching the RFC! Let's update to case insensitive and ship it!